src/test/java/mil/dds/anet/test/utils/ResourceUtilsTest.java

```diff
@@ -175,4 +187,23 @@ private void validateHistory(final Object[][] testData, final boolean checkPerso
   private Instant toInstant(final Object testDate) {
     return testDate == null ? null : Instant.parse((String) testDate);
   }
+
+  @Test
+  void shouldContainClassificationEntries() {
+    then(getAllowedClassifications()).isNotNull().isNotEmpty();
+  }
+
+  @Test
+  void shouldPassWithoutExceptionForExistingClassification() {
+    thenNoException().isThrownBy(() -> assertAllowedClassification("undefined"));
+  }
+
+  @Test
+  void shouldThrowWebApplicationExceptionForUnknownClassification() {
+    thenThrownBy(() -> assertAllowedClassification(RandomString.make()))
```

Comment:
I would avoid using random here, just put something fixed that is invalid.
[Note that we also don't have an explicit dependency on bytebuddy, so this may mysteriously fail when some dependencies are upgraded.]